utils/check_deps.js

```diff
@@ -45,10 +47,10 @@ async function checkDeps() {
 }
 
 async function innerCheckDeps(root, checkDepsFile, checkPackageJson) {
```

Comment:
Do we even pass `checkDepsFile === false`?

Reply:
we do